src/projectController.ts

```diff
@@ -382,7 +385,9 @@ export class ProjectController implements vscode.Disposable {
                 const driver = await project.getCMakeDriverInstance();
                 await driver?.refreshSettings();
             }
-            await ProjectController.checkBuildDirectories(projects);
+            const workspaceContext = DirectoryContext.createForDirectory(folder, new StateManager(this.extensionContext, folder));
```

Comment:
I would think we could get the config without creating a whole DirectoryContext.  Can we?